core/base-service/coalesce-badge.js

```diff
@@ -41,19 +41,34 @@ module.exports = function coalesceBadge(
   defaultBadgeData,
   { category, _cacheLength: defaultCacheSeconds } = {}
 ) {
+  // The "overrideX" naming is based on the endpoint badge, where these are
+  // indeed overrides for values supplied by an endpoint. In the case of the
```

Comment:
In general, "override" here refers to values specified in the query string taking precedence over values provided by the service.

Reply:
Reworded.